addons/hr_timesheet/views/project_task_sharing_views.xml

```diff
@@ -19,7 +19,7 @@
                 <field name="subtask_count" invisible="1"/>
                 <label for="allocated_hours" invisible="not allow_timesheets"/>
                 <div class="text-nowrap" invisible="not allow_timesheets">
-                    <field name="allocated_hours" class="oe_inline" widget="float_time"/>
+                    <field name="allocated_hours" class="oe_inline" widget="timesheet_uom"/>
```

Comment:
To be consistent with the backend view, we could use `timesheet_uom_no_toggle`. 🙂